### PR TITLE
Adding automatic building of Docker Images

### LINKED ISSUE #29: Add Docker image

### PR DESCRIPTION
**Pull request by roock**
## What does this PR do?

[//]:  It adds an Dockerfile to create a docker image containing cloudsplaining. A Github Action to automatically push the Image to the Github Registry is also added. It requires additional configuration to setup a Github Personal Access Token as a Secret in the project (Reference: https://github.com/docker/login-action#github-container-registry)

Closes #29 

## What gif best describes this PR or how it makes you feel?

[//]: # 
grafik

## Completion checklist

- [ ] Additions and changes have unit tests - N/A
- [ ] Python Unit tests, Pylint, security testing, and Integration tests are passing. - N/A
- [ ] Javascript tests are passing (`npm test`) - N/A
- [ ] If the UI contents or JavaScript files have been modified, generate a new example report (`npm build` and `python3 ./utils/generate_example_report.py`) - Generating the example reports works inside the Docker Image
- [ ] The pull request has been appropriately labeled using the provided PR labels

### CODE REVIEW

**Comment by kmcquade** [comment]
Hey @roock - thanks for this PR! Looks great. 

We don't have the GitHub Container registry enabled for our GitHub org yet - but I'm looking into getting that enabled. Should be sometime in November/December. In the meantime, I'll leave this PR opened, and we should be able to merge it then.

**Comment by kmcquade** [comment]
Hey @roock - thanks for this PR! Looks great. 

We don't have the GitHub Container registry enabled for our GitHub org yet - but I'm looking into getting that enabled. Should be sometime in November/December. In the meantime, I'll leave this PR opened, and we should be able to merge it then.

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: docker
+
+on:
+  schedule:
+    - cron: '0 10 * * *' # everyday at 10am
```
Comment (kmcquade) [inline .github/workflows/docker.yml:5]: This runs every day? lol

**Comment by kmcquade** [comment]
@roock is there another option besides using PATs?

**Comment by roock** [comment]
> @roock is there another option besides using PATs?

@kmcquade  I don't think so, although I don't know Github Organizations not well enough to know if there is a workaround.

**Comment by kmcquade** [comment]
Closing this, as it doesn't look like we will get Docker images going through GitHub

**Pull request closed without merging**